airbyte-workers/src/main/java/io/airbyte/workers/WorkerConstants.java

```diff
@@ -28,6 +28,7 @@
 
   public static final String SOURCE_CONFIG_JSON_FILENAME = "source_config.json";
   public static final String DESTINATION_CONFIG_JSON_FILENAME = "destination_config.json";
+  public static final String CUSTOM_ENTRYPOINT_SH = "entrypoint.sh";
```

Comment:
What about
```suggestion
  public static final String DBT_ENTRYPOINT_FILE = "entrypoint.sh";
```

Custom is confusing since it makes me thing this is a custom worker, instead of a custom way of running a specific worker.

Reply:
I also think this can be moved into the DBT class, since this only applies to dbt transformations and all other workers don't have custom entrypoints.